packages/manager/core/generator/app/templates/src/components/layout-helpers/Listing/dataGrid.scss

```diff
@@ -1,15 +1,11 @@
-.datagrid {
-  border: 1px solid #bef1ff;
-  border-collapse: collapse;
+.border-ods-primary-100 {
+  border-color: var(--ods-color-primary-100);
+}
 
-  th {
-    background-color: #bef1ff;
-    padding: 15px;
-  }
+.background-ods-primary-100 {
+  background-color: var(--ods-color-primary-100);
+}
 
-  tr {
-    background-color: #fff;
-    border: 1px solid #bef1ff;
-    border-collapse: collapse;
-  }
+.background-ods-primary-000 {
+  background-color: var(--ods-color-primary-000);
```

Comment:
Don't create utility classes if the file is specific to dataGrid.
If you create utility classes, they should be global.